modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -4825,6 +4825,65 @@ public void testGetValuesList() throws Exception {
 			24, values3, valuesList.get(2), selectedObjectFieldNames);
 	}
 
+	@FeatureFlag("LPD-53981")
+	@Test
+	public void testMoveObjectEntryToTrashWithObjectAction() throws Exception {
+		_objectActionLocalService.addObjectAction(
```

Comment:
It would be nice if we could make `ObjectEntryLocalServiceTest#_addObjectAction` method more flexible to reuse it (new parameters for `objectActionExecutorKey` and `parametersUnicodeProperties`). What do you think?

Reply:
Just done!